test/conformance/testdata/conformance.txt

```diff
@@ -147,6 +147,9 @@ test/e2e/common/projected_secret.go: "should be consumable from pods in volume w
 test/e2e/common/projected_secret.go: "should be consumable in multiple volumes in a pod"
 test/e2e/common/projected_secret.go: "optional updates should be reflected in volume"
 test/e2e/common/runtime.go: "should run with the expected status"
+test/e2e/common/runtime.go: "should report termination message  from log output if TerminationMessagePolicy FallbackToLogsOnError is set"
```

Comment:
Is it fine to not contain [LinuxOnly] in this list?
When I saw this list at first time, I was curious what was duplicated spaces between message and log..

Reply:
Yes. Nice catch. All tags i.e. [Conformance], [NodeConformance], [Slow], [LinuxOnly] etc are filtered out but I feel `[LinuxOnly]/[WindowsOnly]` tag should be part of conformance.txt because I see many e2e has the same name but has different tags and implementations. One for Linux and another for Windows.
e.g.
```
It("should be able to pull image from docker hub [LinuxOnly] [NodeConformance]", func() {
    image := "alpine:3.7"
    imagePullTest(image, false, v1.PodRunning, false, false)
})

It("should be able to pull image from docker hub [NodeConformance]", func() {
    framework.SkipUnlessNodeOSDistroIs("windows")
    // TODO(claudiub): Switch to nanoserver image manifest list.
    image := "e2eteam/busybox:1.29"
    imagePullTest(image, false, v1.PodRunning, false, true)
})
```
>When I saw this list for the first time, I was curious what was duplicated spaces between message and log.

only tags are filtered out but not associated white space by conformance [walker](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/walk.go) :(
`bazel build //test/conformance:list_conformance_tests`

Reply:
Windows tests should not currently be part of this list.